tests/e2e/Services/Health/HealthCustomServerTest.php

```diff
@@ -459,14 +459,14 @@ public function testCertificateValidity(): array
         $this->assertIsInt($response['body']['validFrom']);
         $this->assertIsInt($response['body']['validTo']);
 
-        $response = $this->client->call(Client::METHOD_GET, '/health/certificate?domain=appwrite.io', array_merge([
+        $response = $this->client->call(Client::METHOD_GET, '/health/certificate?domain=letsencrypt.org', array_merge([
```

Comment:
This test failed because appwrite.io changed their issuer from letsencrypt to Google